book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals.push(rental)
+  end
```

Comment:
- You have not properly implemented the has-many relationship between the `Book` class and the `Rental` class. Remember a `book` can have many rentals i.e the add_rental method should look like this
```
def add_rental( date, person)
   Rental.new(date, self, person)
end
```